calliope/core/preprocess/checks.py

```diff
@@ -334,10 +333,11 @@ def _check_tech(model_run, tech_id, tech_config, loc_id, model_warnings, errors,
     # If the technology is supply_plus, check if it has storage_cap_max. If yes, it needs charge rate
     if model_run.techs[tech_id].essentials.parent == 'supply_plus':
         if (any(['storage_cap_' in k for k in tech_config.constraints.keys()])
-            and 'charge_rate' not in tech_config.constraints.keys()):
+            and 'charge_rate' not in tech_config.constraints.keys()
```

Comment:
Any agreement on how to handle this @brynpickering, @sjpfenninger  ? As far as I understand, this is considered as an error only since recently, but Calliope's code base is full of this -- now assumed -- error.

Reply:
Unfortunately our codebase is not consistent, but generally we'd like to see line break before binary operator rather than ine break after binary operator. I've added W503 to the ignore list in the flake8 config on master - should have done that a while ago (hound will only re-run if you push another commit to this branch though).

Reply:
Thanks! Will merge and update after Bryn's review.